### ISSUE TITLE
Rp fails on wrangler due to sbatch change?

### ISSUE THREAD

**Issue by georgeha** [radical-cybertools/radical.pilot#1621]
```
new session: [rp.session.one.george.017670.0000]                               \
database   : [mongodb://rp:rp@ds015335.mlab.com:15335/rp]                     ok
create pilot manager                                                          ok
submit 1 pilot(s)
        .                                                                     ok
create unit manager                                                           ok
add 1 pilot(s)                                                                ok
submit 10 unit(s)
        ..........                                                            ok
wait for 10 unit(s)
        /2018-05-19 22:09:58,246: radical.saga.cpi    : pmgr.0000.launching.0           : Thread-2       : ERROR   : NoSuccess: Couldn't get job id from submitted job! sbatch output:

---------------------------------------------------------------
          Welcome to the Wrangler Supercomputer
---------------------------------------------------------------

--> Submission error: please define total node count with the "-N" option
removed ‘tmp_PUbzu0.slurm’

----------                                                            ok
close unit manager                                                            ok
close pilot manager                                                            \
wait for 1 pilot(s)
                                                                         timeout
                                                                              ok
+ rp.session.one.george.017670.0000 (json)
- pilot.0000 (profiles)
- pilot.0000 (logfiles)
session lifetime: 32.2s                                                       ok

```

I am running the 00_getting_started.py example. Here is my stack:

```
 python               : 2.7.14
  pythonpath           :
  virtualenv           : /home/george/ve

  radical.analytics    : v0.45.2-101-g8358b08@devel
  radical.pilot        : 0.47-0.47-14-gb1dff8bd@devel
  radical.utils        : 0.47.4
  saga                 : 0.47-v0.46-48-g5825f1b0@devel
```

**Comment by andre-merzky** [comment]
A fix has been released in SAGA `v0.47.5`

**Comment by georgeha** [comment]
thanks andre for the quick fix :)
